geo/Geoserver.py

```diff
@@ -1520,33 +1868,13 @@ def delete_coveragestore(
         except Exception as e:
             return "Error: {}".format(e)
 
-    def delete_style(self, style_name: str, workspace: Optional[str] = None):
-        """
-
-        Parameters
-        ----------
-        style_name : str
-        workspace : str, optional
-        """
-        try:
-            payload = {"recurse": "true"}
-            url = "{}/rest/workspaces/{}/styles/{}".format(
-                self.service_url, workspace, style_name
-            )
-            if workspace is None:
-                url = "{}/rest/styles/{}".format(self.service_url, style_name)
-
-            r = requests.delete(url, auth=(
-                self.username, self.password), params=payload)
-
-            if r.status_code == 200:
-                return "Status code: {}, delete style".format(r.status_code)
+    """
+    _______________________________________________________________________________________________
 
-            else:
-                raise Exception("Error: {} {}".format(r.status_code, r.content))
+        USERS AND USERGROUPS
```

Comment:
See previous comments
```suggestion
    # Users and Usergroups
```